src/main/java/duke/Task.java

```diff
@@ -0,0 +1,90 @@
+package duke;
+
+import java.util.ArrayList;
+
+/**
+ * A general task class that handles the status of any task and methods to add a task to the task list.
+ */
+
+public class Task {
+    private String description;
+    private Boolean isDone;
+
+    /**
+     * Constructor for a task with a name
+     * @param name
+     */
+    public Task(String name) {
+        this.description = name;
+        this.isDone = false;
+    }
+
+    /**
+     * Overloaded constructor for a task with a name
+     * @param name
+     * @param isDone if the task is completed
+     */
+    public Task(String name, boolean isDone) {
+        this.description = name;
+        this.isDone = isDone;
+    }
+
+    public boolean findTask(String name) {
+        return description.contains(name);
+    }
+
+    /**
```

Comment:
To quote the coding standard, "In method header comments, the first sentence should start in the form Returns ..., Sends ..., Adds ... (not Return or Returnning etc.)", so something like "Adds tasks from given string format." will be better, also note the capitalisation and full stop at the end.